package.json

```
@@ -18,10 +18,7 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onLanguage:matlab",
-		"onCommand:matlab-interactive-terminal.openMatlabTerminal",
-		"onCommand:matlab-interactive-terminal.runMatlabScript",
-		"onCommand:matlab-interactive-terminal.runMatlabSelection"
+		"*"
```

I don't like that so much, but if there's no choice I guess this would be better:
```suggestion
		"onStartupFinished"
```